### PR TITLE
add healthinsights to Test-SampleMetadata script

### PR DESCRIPTION
**Pull request by asaflevi-ms**
related to csharp PR: https://github.com/Azure/azure-sdk-for-net/pull/34812

### CODE REVIEW

File: eng/common/scripts/Test-SampleMetadata.ps1 [hunk 1]
```diff
@@ -202,6 +202,8 @@ begin {
         "azure-genomics",
         "azure-hdinsight",
         "azure-hdinsight-rserver",
+        "azure-health-insights-cancerprofiling",
+		"azure-health-insights-clinicalmatching",
```
Comment (JimSuplizio) [inline eng/common/scripts/Test-SampleMetadata.ps1:206]: nit: spacing

**Review by JimSuplizio — APPROVED** [review]
I'm okay with the change but please adjust the spacing before merging.

File: eng/common/scripts/Test-SampleMetadata.ps1 [hunk 2]
```diff
@@ -202,6 +202,8 @@ begin {
         "azure-genomics",
         "azure-hdinsight",
         "azure-hdinsight-rserver",
+        "azure-health-insights-cancerprofiling",
```
Comment (weshaggard) [inline eng/common/scripts/Test-SampleMetadata.ps1:205]: Those doc slugs aren't currently in the list supported by the docs team at https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main so adding them here isn't going to help your docs publishing process.

Comment (weshaggard) [reply eng/common/scripts/Test-SampleMetadata.ps1:205]: It does look like azure-health-insights is supported and prehaps you should simply add and use that one.

Comment (asaflevi-ms) [reply eng/common/scripts/Test-SampleMetadata.ps1]: thanks
I updated the product slug to azure-health-insights.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**